lib/plugins/aws/deploy/compile/events/schedule/tests/index.js

```diff
@@ -101,6 +101,33 @@ describe('AwsCompileScheduledEvents', () => {
       ).to.equal('AWS::Lambda::Permission');
     });
 
+    it('should respect input and inputPath variables', () => {
+      awsCompileScheduledEvents.serverless.service.functions = {
+        first: {
+          events: [
+            {
+              schedule: {
+                rate: 'rate(10 minutes)',
+                enabled: false,
+                input: '{}',
```

Comment:
I would also test here some case with real/truthy data (e.g. `input: '{"foo":"bar"}'` and, as separate case,  `inputPath: '$.stageVariables'`.

Also it would great to test the case where we pass the input as an object (in case you think it makes sense to support that syntax in the YAML as per previous comments).